census/customestimator/trainer/model.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
not sure if we should update the copyright year here.  someone told me it should be kept as the first published date.